src/commands.ts

```diff
@@ -476,6 +478,28 @@ async function deployFunction(
   }
 }
 
+export async function deleteProjectHandler(projectId : string, forced : boolean) {
+  // show prompt if no project id is selected
+  if (typeof projectId === 'string' && projectId.trim().length === 0) {
+    console.log('No project ID specified, select an ID to delete from this list:')
+    await listProjects();
```

Comment:
Here after you list all projects, you should give another prompt to the user to put the ID of what project he wants to delete

Reply:
good point, I've added this functionality.